ui/src/app/core-ui/static-properties/static-property.component.css

```diff
@@ -17,25 +17,25 @@
  */
 
 .example-full-width {
-  width: 100%;
+    width: 100%;
 }
 .staticProperty {
-  /* display: block !important;
+    /* display: block !important;
```

Comment:
can be removed?! @dominikriemer @smlabt

Reply:
Could you please confirm this? @dominikriemer @smlabt

Reply:
@aamirxshaikh the uncommented section can be removed, thanks!

Reply:
@dominikriemer Are you referring to the entire staticProperty selector?

```
.staticProperty {
    /* display: block !important;
  width: 100%;
  height: 100%;
  margin-right: 10px; */
}
```

Reply:
@aamirxshaikh Yes in this case you can remove the entire class ".staticProperty", because otherwise it would be empty afterwards